aten/src/ATen/native/cuda/CUDAUnaryOps.cpp

```diff
@@ -4,6 +4,30 @@ namespace at { namespace native {
 
 // These are just forwarding stubs
 
+Tensor _contiguous_cuda(const Tensor& self) {
+  return self.type()._th_contiguous(self);
+}
+
+Tensor& _frac_out_cuda(Tensor& result, const Tensor& self) {
+  result.resize_(self.sizes());
+  return at::_th_frac_out(result, self);
+}
+
+Tensor& _sigmoid_out_cuda(Tensor& result, const Tensor& self) {
+  result.resize_(self.sizes());
+  return at::_th_sigmoid_out(result, self);
+}
+
+Tensor& _neg_out_cuda(Tensor& result, const Tensor& self) {
+  result.resize_(self.sizes());
+  return at::_th_neg_out(result, self);
+}
+
+Tensor& _reciprocal_out_cuda(Tensor& result, const Tensor& self) {
+  result.resize_(self.sizes());
+  return at::_th_reciprocal_out(result, self);
+}
```

Comment:
Any reason why those can't use the macros like all other functions below?

Reply:
Will do this while vectorizing.